app/views/task/new.html.erb

```diff
@@ -4,14 +4,11 @@
 </header>
 
 <%= form_for @task do |f| %>
-  <fieldset>
-    <legend>So hurry up and add a book:</legend>
-    <%= f.label :taskname, "Book Title" %>
-    <%= f.text_field :taskname %>
-    <%= f.label :description, "Details" %>
-    <%= f.text_field :description %>
-  </fieldset>
 
-  <%= f.submit 'Submit' %>
+  <%= render partial: 'form', locals: { f: f } %>
 
 <% end %>
+
+<ul class = "mininav">
```

Comment:
I know we literally just learned about view helpers - but this would be a great place to use one since you use this markup more than once!